Src/BootCamp.Chapter/Checks.cs

```diff
@@ -9,31 +7,192 @@ public static class Checks
     {
         public static string FindMostPoorPerson(string[] peopleAndBalances)
         {
-            return "";
+            if (Testers.IsThisStringArrayValid(peopleAndBalances))
+            {
+                return "N/A.";
+            }
+            string poorestName = "";
+            decimal amount = 0;
+            int personCount = 1;
+
+            for (int i = 0; i < peopleAndBalances.Length; i++)
```

Comment:
All the string concatination from a loop should be done using StringBuilder. Please refactor.